src/details/ArborX_MinimumSpanningTree.hpp

```diff
@@ -214,7 +215,10 @@ struct FindComponentNearestNeighbors
     DirectedEdge current_best{};
 
     auto const n = _bvh.size();
-    auto &radius = _radii(component - n + 1);
+
+    // Use a reference for shared radii, and a value otherwise.
```

Comment:
```suggestion
    // Use a reference for shared radii, and a copy otherwise.
```

Reply:
Did we forget to implement the [CI skip]?

Reply:
I did